os_reststack_manager/app/__init__.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+import os
+import os_reststack_manager.config as CONF
+import logging
+
+from flask import Flask
+from flask.ext.sqlalchemy import SQLAlchemy
+from lib.setup_tenant import parse_config
+
+
+app = Flask(__name__)
+app.config.from_object('os_reststack_manager.config')
+
+# DB setup
+db = SQLAlchemy(app)
+db.create_all()
+
+credentials = parse_config(CONF.CREDENTIALS)
+
+
+class Tenant(db.Model):
```

Comment:
This code needs to be between lines 16 and 17 or else the db won't have the right schema on creation.